docs/conversion/sakai_12_mysql_conversion.sql

```diff
@@ -618,3 +618,125 @@ DROP TABLE IF EXISTS SITEASSOC_CONTEXT_ASSOCIATIO;
 
 --END SAK-33896 
 
+
+CREATE TABLE ASN_ASSIGNMENT (
+    ASSIGNMENT_ID                  VARCHAR(36) NOT NULL,
+    ALLOW_ATTACHMENTS              BIT,
+    ALLOW_PEER_ASSESSMENT          BIT,
+    AUTHOR                         VARCHAR(36),
```

Comment:
this number (36) should match sakai_user_id_map id size, right?

Reply:
ASSIGNMENT_ID is correct this can't be created by users.